app/templates/admin/createEvent.html

```diff
@@ -144,41 +144,25 @@ <h1 id="title">{{page_title}}</h1>
             <textarea rows="5" cols="72" class="form-control" id='inputEventDescription' placeholder="Enter event description" name="description" required>{{eventData.description}}</textarea>
           </div>
-          <div class='form-group'>
-            <label class="form-label" for='inputEventFacilitators'><strong>Facilitators</strong></label>
-            <input class="form-control" type="text" id="hiddenFacilitatorArray" name="facilitators" value="" hidden>
-            <input
-              id="eventFacilitator"
-              class="form-control"
-              type="search"
-              placeholder="Search facilitator"
-              value="" />
-            <table class="table mb-3" id="facilitatorTable" name="facilitatorTable">
-              <thead>
-                <tr>
-                </tr>
-              </thead>
-              <tbody>
-                <tr hidden>
-                  <td>
-                    <p class="facilitatorName mt-3" required></p>
-                  </td>
-                  <td><button id="remove" type="button" class="mt-3 btn btn-danger">Remove</button></td>
-
-                </tr>
-                {% for instructor in courseInstructor %}
-                <tr>
-                  <td>
-                    <p class="facilitatorName">{{instructor.user.firstName}} {{instructor.user.lastName}} ({{instructor.user}})</p>
-                  </td>
-                  <td>
-                    <button id="remove" type="button" class="btn btn-danger">Remove</button>
-                  </td>
-                </tr>
-                {% endfor %}
-              </tbody>
-            </table>
-           </div>
+          {% if contactInfo %}
+              <div class="form-group mb-4">
+                <label class="form-label" for='inputContactName'><strong>Contact Name</strong></label>
+                <input class="form-control" id='inputContactName' value="{{contactInfo['contactName']}}" name="contactName" required>
+              </div>
+              <div class="form-group mb-4">
+                <label class="form-label" for='inputContactEmail'><strong>Contact Email</strong></label>
+                <input class="form-control" id='inputContactEmail' value="{{contactInfo['contactEmail']}}" name="contactEmail" required>
+              </div>
+          {% else %}
+              <div class="form-group mb-4">
+                <label class="form-label" for='inputContactName'><strong>Contact Name</strong></label>
+                <input class="form-control" id='inputContactName' value="" placeholder="Enter name" name="contactName" required>
+              </div>
+              <div class="form-group mb-4">
+                <label class="form-label" for='inputContactEmail'><strong>Contact Email</strong></label>
+                <input class="form-control" id='inputContactEmail' value="" placeholder="Enter email" name="contactEmail" required>
+              </div>
+          {% endif %}
```

Comment:
We definitely don't want to repeat all these blocks. Just make sure contact_info exists, and you can just use that dictionary. Or better yet, just use eventData like everything else

Reply:
resolved